synapse/lib/node.py

```diff
@@ -252,20 +257,17 @@ def get(self, name):
         return node.get(text)
 
     def pop(self, name, init=False):
+
```

Comment:
Callers of this function have to check the return value of None vs. False to know if a property was popped'

Reply:
noticed that.  was thinking about making it return the value ( or s_common.novalu ) so we could avoid
```
oldv = node.get('foo')
node.pop('foo')
dostuff(oldv)
```
Leaving it for now, but it def needs a tweak.